File: examples/event/consume/go/consumeFromATopic.go
package example

import (
	"fmt"
	"os"

	"github.com/micro/services/clients/go/event"
)

// Consume events from a given topic.
func ConsumeFromAtopic() {
	eventService := event.NewEventService(os.Getenv("MICRO_API_TOKEN"))
	rsp, err := eventService.Consume(&event.ConsumeRequest{
		Topic: "user",
	})
	fmt.Println(rsp, err)
}

File: examples/event/consume/node/consumeFromATopic.js
const { EventService } = require("micro-js-client/event");

// Consume events from a given topic.
async function consumeFromAtopic() {
  let eventService = new EventService(process.env.MICRO_API_TOKEN);
  let rsp = await eventService.consume({
    topic: "user",
  });
  console.log(rsp);
}

consumeFromAtopic();
